frontend/src/pages/ChainEdit.js

```diff
@@ -78,11 +86,22 @@ const ChainEdit = () => {
   };
 
   //refactor db data from array to obj
-  const mapCat = (array) => {
+  const mapGenders = (array) => {
     return {
       men: array.includes("men"),
-      woman: array.includes("woman"),
-      mix: array.includes("mix"),
+      woman: array.includes("women"),
+      children: array.includes("children"),
+    };
+  };
+
+  const mapSizes = (array) => {
+    return {
+      xs: array.includes("xs"),
+      s: array.includes("s"),
+      m: array.includes("m"),
+      l: array.includes("l"),
+      xl: array.includes("xl"),
+      xxl: array.includes("xxl"),
```

Comment:
mapSizes and mapGenders should be probably be using some global definition of what values are allowed in the categories. The current setup allows for too many mistakes.